src/main/java/uk/gov/hmcts/dts/fact/entity/Court.java

```diff
@@ -106,6 +106,10 @@
     @OneToMany(mappedBy = COURT_STRING, orphanRemoval = true)
     private List<CourtPostcode> courtPostcodes;
 
+    @OneToMany(mappedBy = COURT_STRING, orphanRemoval = true)
+    @OrderBy("sort")
```

Comment:
Order by sort is cool, do you want to also return them in an ascending or desc order, or does it not matter as much? 

I see further down you get it by ID desc when inserting, one to bear in mind, not quite sure how it will be done on the frontend, or if it matters!

Reply:
The front end wont sort anything. The backend will return the values as sorted in the database. We should try and avoid bespoke logic on the front end. Eventually the admin portal will have a new tab. In this admin tab, they can move up and down (the logic implemented by Andrew). The list will send a sorted list to the backend and they will be saved and sorted accordingly in the front end too